tools/version.go

```diff
@@ -0,0 +1,7 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package tools
+
+const DefaultBeatVersion = "8.4.0"
```

Comment:
We can automate the release branch creation and version update for this project if it is part of the Unified Release